src/api/app/assets/stylesheets/webui2/main.scss

```diff
@@ -0,0 +1,9 @@
+#personal-navigation ul.nav >li.nav-item > a {
+  padding-left: 0;
+  padding-right: 0;
```

Comment:
There are [bootstrap classes](https://getbootstrap.com/docs/4.1/utilities/spacing/) for this.